cmd/app-example/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	abs "github.com/microsoft/kiota-abstractions-go"
+	"github.com/octokit/go-sdk/pkg"
+	"github.com/octokit/go-sdk/pkg/github/installation"
+)
+
+func main() {
+	installationID, err := strconv.ParseInt(os.Getenv("INSTALLATION_ID"), 10, 64)
+	if err != nil {
+		log.Fatalf("error parsing installation ID from string to int64: %v", err)
+	}
+
+	client, err := pkg.NewApiClient(
```

Comment:
The fluent syntax here makes me want to consider how we might be able to propagate that to other SDKs - it feels really good.